File: LC389.go
func findTheDifference(s string, t string) byte {
    var count [26]int;
    for i:=0; i < len(s); i++{
        count[s[i]-'a']++;
    }
    for i:=0; i < len(t); i++{
        count[t[i]-'a']--;
        if count[t[i]-'a'] < 0{
            return t[i];
        }
    }
    return t[0];
}

File: src/LC389.java

    class Solution {
        public char findTheDifference(String s, String t) {
            int[] sarr = new int[26];
            for(int i=0;i< s.length();i++)
            {
                sarr[s.charAt(i)-'a']++;
            }
            int[] tarr = new int[26];
            for(int i=0;i< t.length();i++)
            {
                tarr[t.charAt(i)-'a']++;
            }
            for(int i=0;i<26;i++)
            {
                if(sarr[i]!=tarr[i])
                    return (char)('a'+i);
            }
            return 'a';
        }
    }